samples-generator/bin/3d-tiles-samples-generator.js

```diff
@@ -1478,7 +1522,7 @@ function createTilesetRefinementMix() {
             geometricError : smallGeometricError,
             refine : 'ADD',
             content : {
-                url : 'parent.b3dm',
+                uri : 'parent.b3dm',
```

Comment:
There are a few other samples that need to be updated like these: `createTileset`, `createTilesetEmptyRoot`, `createTilesetOfTilesets`, `createTilesetWithExternalResources`,